localnet/poktrolld/config/relayminer_config_full_example.yaml

```diff
@@ -0,0 +1,108 @@
+# TODO_CONSIDERATION: We don't need this now, but it would be beneficial if the
+# logic handling this config file could be designed in such a way that it allows for
+# "hot" config changes in the future, meaning changes without restarting a process.
+# This would be useful for adding a proxy or a supplier without interrupting the service.
+
+# Name of the key (in the keyring) to sign transactions
+signing_key_name: supplier1
+# Relative path (on the relayminer's machine) to where the data backing
+# SMT KV store exists on disk
+smt_store_path: smt_stores
+
+pocket_node:
+  # Pocket node URL that exposes CometBFT JSON-RPC API.
+  # This can be used by the Cosmos client SDK, event subscriptions, etc...
+  query_node_rpc_url: tcp://sequencer-poktroll-sequencer:36657
+  # Pocket node URL that exposes the Cosmos gRPC service, dedicated to querying purposes.
+  # If unspecified, defaults to `tx_node_grpc_url`.
+  query_node_grpc_url: tcp://sequencer-poktroll-sequencer:36658
+  # Pocket node URL that exposes the Cosmos gRPC service.
+  tx_node_grpc_url: tcp://sequencer-poktroll-sequencer:36658
+
+# Proxies are endpoints that expose different suppliers to the internet.
+proxies:
+    # Name of the proxy. It will be used to reference in a supplier. Must be unique.
+    # Required.
+    # TODO_CONSIDERATION: if we enforce DNS compliant names, it can potentially
+    # become handy in the future.
+    # More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names
+  - name: http-example
```

Comment:
Can you add a `TODO_CONSIDERATION: Rename to supplier_name`

Reply:
It would rather be `proxy_name` since it identifies a `proxy`